configurate-core/src/test/java/ninja/leaping/configurate/SimpleConfigurationNodeTest.java

```diff
@@ -16,11 +16,18 @@
  */
 package ninja.leaping.configurate;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
Static imports should be after all other imports.

I also think it's probably ok to use wildcard import statements in the test classes.